data/strings/strings.txt

```diff
@@ -16382,6 +16382,143 @@
+    de = Organic Maps unterstützt nicht alle Kategorien. So können z.B. Städte, Bahnhöfe, Straßen, Seen oder Gebäude nicht hinzugefügt werden. Bitte füge solche Orte direkt über osm.org hinzu (verwende die Schaltfläche Bearbeiten in der oberen Leiste) oder verwende eine spezielle OSM-Bearbeitungs-App.
+    el = Οι οργανικοί χάρτες δεν υποστηρίζουν όλες τις κατηγορίες. Π.χ. δεν μπορούν να προστεθούν πόλεις, σιδηροδρομικοί σταθμοί, δρόμοι, λίμνες ή κτίρια. Παρακαλούμε προσθέστε τέτοια χαρακτηριστικά απευθείας μέσω του osm.org (χρησιμοποιήστε το κουμπί Επεξεργασία στην επάνω μπάρα) ή χρησιμοποιήστε μια εξειδικευμένη εφαρμογή επεξεργασίας OSM.
+    es = Organic Maps no admite todas las categorías. Por ejemplo, no se pueden añadir ciudades, estaciones de ferrocarril, carreteras, lagos o edificios. Por favor, añade estas características directamente a través de osm.org (utiliza el botón Editar de la barra superior) o utiliza una aplicación de edición OSM especializada.
+    es-MX = Organic Maps no admite todas las categorías. P.ej. No se pueden agregar ciudades, estaciones de tren, carreteras, lagos o edificios. Agregue dichas funciones directamente a través de osm.org (use el botón Editar en la barra superior) o use una aplicación de edición especializada de OSM.
+    et = Organic Maps ei toeta kõiki kategooriaid. Näiteks ei saa lisada linnu, raudteejaamu, teid, järvi või hooneid. Palun lisage sellised funktsioonid otse osm.org kaudu (kasutage ülemise riba nuppu Edit) või kasutage spetsiaalset OSMi redigeerimisrakendust.
+    eu = Organic Maps-ek ez ditu kategoria guztiak onartzen. Adib. ezin dira gehitu hiriak, tren geltokiak, errepideak, lakuak edo eraikinak. Mesedez, gehitu funtzio horiek zuzenean osm.org bidez (erabili goiko barrako Editatu botoia) edo erabili OSM editatzeko aplikazio espezializatu bat.
+    fa = نقشه های ارگانیک از همه دسته ها پشتیبانی نمی کند. به عنوان مثال. شهرها، ایستگاه های راه آهن، جاده ها، دریاچه ها یا ساختمان ها را نمی توان اضافه کرد. لطفاً چنین ویژگی هایی را مستقیماً از طریق osm.org اضافه کنید (از دکمه ویرایش در نوار بالا استفاده کنید) یا از یک برنامه ویرایش تخصصی OSM استفاده کنید.
+    fi = Organic Maps ei tue kaikkia luokkia. Esimerkiksi kaupunkeja, rautatieasemia, teitä, järviä tai rakennuksia ei voi lisätä. Lisää tällaiset ominaisuudet suoraan osm.org-sivuston kautta (käytä yläpalkin Muokkaa-painiketta) tai käytä erikoistunutta OSM-editointisovellusta.
+    fr = Organic Maps ne prend pas en charge toutes les catégories. Par exemple, les villes, les gares, les routes, les lacs ou les bâtiments ne peuvent pas être ajoutés. Ajoute ces éléments directement via osm.org (utilise le bouton Modifier dans la barre supérieure) ou utilise une application d'édition OSM spécialisée.
+    he = מפות אורגניות לא תומכות בכל הקטגוריות. לְמָשָׁל. לא ניתן להוסיף ערים, תחנות רכבת, כבישים, אגמים או מבנים. אנא הוסף תכונות כאלה ישירות דרך osm.org (השתמש בכפתור העריכה בסרגל העליון) או השתמש באפליקציית עריכה מיוחדת של OSM.
```

Comment:
```suggestion
    he = האפליקציה Organic Maps יכולה להוסיף רק נקודות, לא קווים או צורות. למשל, לא ניתן להוסיף כבישים, מבנים, ערים, תחנות רכבת, או אגמים. ניתן להוסיף תכונות כאלה ישירות דרך osm.org (השתמשו בכפתור העריכה בסרגל העליון באתר) או דרך אפליקציית עריכה ייעודית של OSM.
```

I based this on the suggested rephrasing I posted above (which was based on pastk's suggestion) not the current commit. Here's a back-translation to English (via google translate):

> The Organic Maps app can only add points, not lines or shapes. For example, you cannot add roads, buildings, cities, train stations, or lakes. Such features can be added directly through osm.org (use the edit button in the top bar of the site) or through a dedicated OSM editor application.